src/Microsoft.Windows.Compatibility/src/PACKAGE.md

```diff
@@ -0,0 +1,92 @@
+# Microsoft.Windows.Compatibility
+
+The `Microsoft.Windows.Compatibility` package provides Windows-specific APIs to help you port your .NET Framework applications to .NET Core 2.0+, .NET 5+ or .NET Standard. This package offers a smoother transition for those looking to modernize their applications without losing access to familiar Windows functionalities.
+
+## Getting Started
```

Comment:
Move the link to nuget.org to this section?

Reply:
At that point, aren't folks already on that page?